graphql/schema.ts

```diff
@@ -75,3 +77,25 @@ export const StackCategoryEnum = {
 } as const
 
 export type StackCategory = keyof typeof StackCategoryEnum;
+
+export type Image = {
```

Comment:
This is starting to scale, so in order to maintain a better contract with the API schema, I'll work to generate TS type definitions using a tool like [GraphQL code generator]